src/core/foundation.hpp

```diff
@@ -108,44 +108,44 @@ struct CBCoro {
 
 namespace chainblocks {
 #ifdef CB_COMPRESSED_STRINGS
-CBOptionalString getCompiledCompressedString(uint32_t crc);
+CHAINBLOCKS_API CBOptionalString getCompiledCompressedString(uint32_t crc);
```

Comment:
don't export - as said before
Also those are internal API which we want to break often